ci-operator/step-registry/mirror-images/icsp-olm-operators/mirror-images-icsp-olm-operators-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+mirror_auths="${SHARED_DIR}/mirror_auths"
+
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy:"
+        cat "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
It's easy for us to check the proxy setting when some failures, and the content will not be display, such as log: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/28223/rehearse-28223-periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-usgov-qe/1529400277721944064/artifacts/installer-rehearse-aws-usgov-qe/mirror-images-icsp-olm-operators/build-log.txt
```yaml
setting the proxy:
export http_proxy=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:3128
export https_proxy=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:3128
export no_proxy="localhost,127.0.0.1"
export HTTP_PROXY=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:3128
export HTTPS_PROXY=http://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX@ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:3128
export NO_PROXY="localhost,127.0.0.1"
```

Reply:
There are many places in the repo which has the same case, even they are more simple like this:
```
if test -f "${SHARED_DIR}/proxy-conf.sh"
then
        # shellcheck disable=SC1090
        source "${SHARED_DIR}/proxy-conf.sh"
fi
```

I am not sure the exposing these info on purpose will violate some security best practice. 
And I do not think it can help users much for trouble-shooting to print out these info in log. Once the job is completed, the failed cluster is removed immediately, even you can see these info, you still can not assure the proxy server is really working there.

Reply:
make sense.